src/playground/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "askscript-playground",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "build": "mkdir -p dist/public && npm run build-assets && npm run build-ts && npm run copy-static-assets",
```

Comment:
How is this going to integrate with the build process of the entire repo?

Reply:
Not sure what you mean. 
monorepo means many packages in one repository, so that code can be shared between them. 
Every package has their own package.json file with their own build instructions. Therefore, there is no single build process as the packages are independent, same as they would be if they were in separate repositories.

Reply:
I'm asking because we don't currently have package-level package.json. Why introduce it here? Is build process of this package different than that of AskQL? How so? If it was the same, then we wouldn't need to add this file. What about releases of the playground? Is this going to NPM?

Reply:
Marcin, Playground frontend is a separate project. It's separate from AskVM and AskScript parser both in terms of a separate codebase and in terms of dependencies.
It has a number of dependencies not needed in any other project, like `imagemin-cli, node-sass, webpack, webpack-cli, @types/express-handlebars` to name a few.
Adding them to the main package.json would create a lot of clutter and a mix of dependencies needed by AskVM and Playground.